### ISSUE TITLE
Missing files for Python 3.12

### ISSUE THREAD

**Issue by mraunak** [tensorflow/io#1966]
Hello @yongtang, it has been some time since TensorFlow-IO had a release for the Windows platform, and there hasn't been a release across all platforms for Python 3.12 either. Are there any immediate plans to have wheels available with a forthcoming release?

**Comment by MatthiasMuller** [comment]
Is there an anticipated release of Windows in any of the upcoming versions?

**Comment by thebaptiste** [comment]
Yes, please add missing x86_64 python 3.12 wheel for release 0.36.0 on pypi : 
tensorflow_io_gcs_filesystem-0.36.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
